test/e2eprovider/server/server_test.go

```diff
@@ -213,3 +213,33 @@ func TestRotation(t *testing.T) {
 		t.Errorf("didn't get expected results: (-want +got):\n%s", diff)
 	}
 }
+
+func TestValidateTokens(t *testing.T) {
+	tests := []struct {
+		name      string
+		tokens    string
+		audiences string
+		wantErr   bool
```

Comment:
Suggest just making 1 passing test and a `TestValidateTokensErrors` for the error case (based on https://github.com/golang/go/wiki/TestComments#table-driven-tests-vs-multiple-test-functions)

Reply:
done!